sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsWarning.cs

```diff
@@ -22,6 +22,12 @@ internal TextAnalyticsWarning(DocumentWarning warning)
             Message = warning.Message;
         }
```

Comment:
Note to self: update so that this delegates to the component constructor and removes duplication.